marklogic-data-hub/src/main/java/com/marklogic/hub/DataHub.java

```diff
@@ -178,16 +178,6 @@
      */
     boolean upgradeHub() throws CantUpgradeException;
 
-    /**
-     * Upgrades the hub based on list of provided updated flows. All flows SHOULD be provided.
-     * The method without params will handle this automatically.
-     * Must be run as a user with sufficient privileges to install a data hub.
-     * @param updatedFlows - the list of the name of the flows you want to update
-     * @return boolean - false if upgrade fails for a reason other than an upgrade exception
-     * @throws CantUpgradeException - should the hub fail to upgrade for incompatibility reasons
-     */
-    boolean upgradeHub(List<String> updatedFlows) throws CantUpgradeException;
```

Comment:
I removed this, as updatedFlows was never used. I know this is part of the public interface for DataHub, but I can't imagine anyone was calling this directly since updatedFlows was ignored.